### PR TITLE
added single-letter flags to commonly used flags

### PR DESCRIPTION
**Pull request by lior-stratoscale**

### CODE REVIEW

**Comment by rom-stratoscale** [comment]
@sergey-stratoscale

File: py/strato/common/log/log2text.py [hunk 1]
```diff
@@ -235,12 +235,12 @@ def updateConfFile(field, value):
     import argparse
     parser = argparse.ArgumentParser()
     parser.add_argument("logFiles", metavar='logFile', nargs='*', help='logfiles to read or - for stdin')
-    parser.add_argument("--noDebug", action='store_true', help='filter out debug messages')
-    parser.add_argument("--relativeTime", action='store_true', help='print relative time, not absolute')
-    parser.add_argument("--noColors", action='store_true', help='force monochromatic output even on a TTY')
-    parser.add_argument(
+    parser.add_argument('-D', "--noDebug", action='store_true', help='filter out debug messages')
```
Comment (sergey-stratoscale) [inline py/strato/common/log/log2text.py]: from my experience, -D usually stands for debug mode, which is the opposite of it. please change it

**Comment by sergey-stratoscale** [comment]
other than that, ship it

**Comment by rom-stratoscale** [comment]
@sergey-stratoscale  did you push it? If so, lets close the PR

File: py/strato/common/log/log2text.py [hunk 2]
```diff
@@ -235,12 +235,12 @@ def updateConfFile(field, value):
     import argparse
     parser = argparse.ArgumentParser()
     parser.add_argument("logFiles", metavar='logFile', nargs='*', help='logfiles to read or - for stdin')
-    parser.add_argument("--noDebug", action='store_true', help='filter out debug messages')
-    parser.add_argument("--relativeTime", action='store_true', help='print relative time, not absolute')
-    parser.add_argument("--noColors", action='store_true', help='force monochromatic output even on a TTY')
-    parser.add_argument(
+    parser.add_argument('-d', "--noDebug", action='store_true', help='filter out debug messages')
```
Comment (adir-stratoscale) [inline py/strato/common/log/log2text.py]: maybe change it to 
{code}
parser.add_argument('-D', "--debug", action='store_false', help='show debug messages')
{code}

**Pull request closed without merging**